app/assets/javascripts/d3/bezier.coffee

```diff
@@ -141,7 +141,7 @@ D3.bezier =
 
       @draw_legend
         svg: @svg
-        series: series
+        series: series.reverse()
```

Comment:
Is it right to make this change only for the bezier chart? I'm not keen on how this makes the bezier chart inconsistent with the others:

<img width="607" alt="screen shot 2017-11-23 at 13 15 26" src="https://user-images.githubusercontent.com/4383/33174550-7ab4fa6c-d050-11e7-9b76-7487050fe9ac.png">

<img width="607" alt="screen shot 2017-11-23 at 13 16 54" src="https://user-images.githubusercontent.com/4383/33174592-9a87342c-d050-11e7-9ed4-ec09dfe2055d.png">

<img width="617" alt="screen shot 2017-11-23 at 13 19 03" src="https://user-images.githubusercontent.com/4383/33174678-ed6d363c-d050-11e7-8c9d-b49cdf92d239.png">